### PR TITLE
Updated documentation in image_ops_impl.py 

### PR DESCRIPTION
**Pull request by srihari-humbarwadi**
Fixed grammatical errors in documentation  image_ops_impl.py

### CODE REVIEW

**Review by mihaimaruseac — APPROVED** [review]

File: tensorflow/python/ops/image_ops_impl.py [hunk 1]
```diff
@@ -2162,7 +2162,7 @@ def is_jpeg(contents, name=None):
     name: A name for the operation (optional)
 
   Returns:
-     A scalar boolean tensor indicating if 'contents' may be a JPEG image.
+     A scalar boolean tensor indicating if 'contents' maybe a JPEG image.
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/image_ops_impl.py]: This should be "may be". Similar in other places downwards

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

File: tensorflow/python/ops/image_ops_impl.py (hunk 1 above)
Comment (srihari-humbarwadi) [reply tensorflow/python/ops/image_ops_impl.py]: @mihaimaruseac made the required changes

File: tensorflow/python/ops/image_ops_impl.py [hunk 2]
```diff
@@ -308,7 +308,7 @@ def fix_image_flip_shape(image, result):
     result: flipped or transformed image
 
   Returns:
-    An image whose shape is at least None,None,None.
+    An image whose shape is at least None, None, None.
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/image_ops_impl.py]: Should be as a tuple, `(None, None, None)`.

File: tensorflow/python/ops/image_ops_impl.py [hunk 3]
```diff
@@ -407,7 +407,7 @@ def _random_flip(image, flip_index, seed, scope_name):
   Args:
     image: 4-D Tensor of shape `[batch, height, width, channels]` or 3-D Tensor
       of shape `[height, width, channels]`.
-    flip_index: Dimension along which to flip image. Vertical: 0, Horizontal: 1
+    flip_index: Dimension along which to flip the image. Vertical: 0, Horizontal: 1
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/image_ops_impl.py]: Please make sure to not pass 80 characters limit on a line. Make sure you run pylint

File: tensorflow/python/ops/image_ops_impl.py [hunk 4]
```diff
@@ -3102,7 +3102,7 @@ def _ssim_helper(x, y, reducer, max_val, compensation=1.0, k1=0.01, k2=0.03):
     compensation: Compensation factor. See above.
     k1: Default value 0.01
     k2: Default value 0.03 (SSIM is less sensitivity to K2 for lower values, so
-      it would be better if we taken the values in range of 0< K2 <0.4).
+      it would be better if we took the values in the range of 0< K2 <0.4).
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/image_ops_impl.py]: `0 < K2 < 0.4` since you fixed spacing on `None,None,None` above

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

File: tensorflow/python/ops/image_ops_impl.py (hunk 2 above)
Comment (srihari-humbarwadi) [reply tensorflow/python/ops/image_ops_impl.py]: made required changes

File: tensorflow/python/ops/image_ops_impl.py (hunk 3 above)
Comment (srihari-humbarwadi) [reply tensorflow/python/ops/image_ops_impl.py]: made required changes

File: tensorflow/python/ops/image_ops_impl.py (hunk 4 above)
Comment (srihari-humbarwadi) [reply tensorflow/python/ops/image_ops_impl.py]: made required changes

File: tensorflow/python/ops/image_ops_impl.py [hunk 5]
```diff
@@ -3177,7 +3179,7 @@ def _ssim_per_channel(img1,
     filter_sigma: Default value 1.5 (width of gaussian filter).
     k1: Default value 0.01
     k2: Default value 0.03 (SSIM is less sensitivity to K2 for lower values, so
-      it would be better if we taken the values in range of 0< K2 <0.4).
+      it would be better if we took the values in the range of 0< K2 <0.4).
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/image_ops_impl.py]: Here too.

File: tensorflow/python/ops/image_ops_impl.py [hunk 6]
```diff
@@ -3284,7 +3286,7 @@ def ssim(img1,
...
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/image_ops_impl.py]: Here too.

File: tensorflow/python/ops/image_ops_impl.py [hunk 7]
```diff
@@ -3347,7 +3349,7 @@ def ssim_multiscale(img1,
...
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/image_ops_impl.py]: Here too.

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

File: tensorflow/python/ops/image_ops_impl.py (hunk 5 above)
Comment (srihari-humbarwadi) [reply tensorflow/python/ops/image_ops_impl.py]: made required changes

File: tensorflow/python/ops/image_ops_impl.py (hunk 6 above)
Comment (srihari-humbarwadi) [reply tensorflow/python/ops/image_ops_impl.py]: made required changes

File: tensorflow/python/ops/image_ops_impl.py (hunk 7 above)
Comment (srihari-humbarwadi) [reply tensorflow/python/ops/image_ops_impl.py]: @mihaimaruseac made required changes

**Review by mihaimaruseac — APPROVED** [review]

**Comment by rthadur** [comment]
@srihari-humbarwadi can you please fix the sanity errors ?

**Comment by srihari-humbarwadi** [comment]
```
5. do_buildifier: buildifier check
  FAIL
```
@rthadur Looks like`buildifier` failed. But I did not change any build rules, don't think so the files i made changes in are failing the check, can you please confirm?

**Comment by rthadur** [comment]
np, let me try to pull this in.

**Comment by srihari-humbarwadi** [comment]
Thank You!

**Comment by rthadur** [comment]
it got merged internally , waiting for auto-merge to happen.

**Pull request merged**